google/api_core/gapic_v1/method.py

```diff
@@ -31,9 +31,6 @@
 class _MethodDefault(enum.Enum):
     # Uses enum so that pytype/mypy knows that this is the only possible value.
     # https://stackoverflow.com/a/60605919/101923
-    #
-    # Literal[_DEFAULT_VALUE] is an alternative, but only added in Python 3.8.
-    # https://docs.python.org/3/library/typing.html#typing.Literal
```

Comment:
it looks like this comment is saying an improvement is available to us now that we're dropping 3.7. Is that something we'd want to consider, or should we stick with the existing implementation?